.golangci.reference.yml

```diff
@@ -356,13 +356,15 @@ linters-settings:
       - 'example.com/package.ExampleStruct'
 
   exhaustruct:
-    # List of regular expressions to match struct packages and names.
+    # List of regular expressions to match struct packages and their names.
+    # Regular expressions must match complete canonical struct package/name/structname.
     # If this list is empty, all structs are tested.
     # Default: []
     include:
       - '.*\.Test'
       - 'example\.com/package\.ExampleStruct[\d]{1,2}'
-    # List of regular expressions to exclude struct packages and names from check.
+    # List of regular expressions to exclude struct packages and their names from checks.
+    # Regular expressions must match complete canonical struct package/name/structname.
     # Default: []
     exclude:
       - 'cobra\.Command$'
```

Comment:
Actually - now I look at it - I don't think `'cobra\.Command$'` could ever be a valid match either?!

Reply:
Just added 7df401d2b58e11518570770b7173204c7c84221f to fix that fact.